sound/soc/codecs/rt712-sdca.c

```diff
@@ -1652,6 +1652,11 @@ int rt712_sdca_init(struct device *dev, struct regmap *regmap,
 	if (ret < 0)
 		return ret;
 
+	ret =  devm_snd_soc_register_component(dev,
+				&soc_sdca_dev_rt712_dmic, rt712_sdca_dmic_dai, ARRAY_SIZE(rt712_sdca_dmic_dai));
+	if (ret < 0)
+		return ret;
+
```

Comment:
@plbossart If we could detect the RT712VB (a mic function ACPI) before rt712_sdca_init, the dmic component registration would be nice to move here.

Reply:
we can, the function already exists but I can't figure out how to get the Kconfig right...